Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -208,6 +210,31 @@ def check_elisp_root(share, name)
     EOS
   end
 
+  def check_linkage
+    keg = Keg.new(formula.prefix)
```

Comment:
The failure for when a formula is not installed should be fixable with `return unless formula.prefix.directory?` prior to this line, just like in the `check_openssl_links` method.